### PR TITLE
Experimental/new hero ux

### PR DESCRIPTION
**Pull request by yasmineevjen**
- Added Material Icons instructions link and overline padding.
- Made syntax line numbers unselectable
- Fixed hero width, containing it within the page
- Changed chip color
- Changed options labels
- Changed padding and margins

TODO Fix mobile design

### CODE REVIEW

**Comment by yasmineevjen** [comment]
I signed it!

File: package-lock.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
       "integrity": "sha512-oouEibCbHMVdZSDlJBO6bZmID/zA/G/Qx3H1d3rSNPTD+L8UNKvCat7aKWSJ74zYbm5zWGh0GQN0hKj8zYFTCg==",
       "dev": true,
       "requires": {
-        "regenerator-runtime": "^0.12.0"
+        "regenerator-runtime": "0.12.1"
```
Comment (moog16) [inline package-lock.json]: revert these change
